.github/CODEOWNERS

```diff
@@ -427,6 +427,7 @@
 /core/templates/dev/head/pages/profile-page/ @kevinlee12
 /core/templates/dev/head/services/UserService*.ts @kevinlee12
 
+/core/templates/dev/head/services/services.constants.ts @kevinlee12 @seanlip
```

Comment:
Just as a note, I don't need to own this :)

Reply:
I added them as I wasn't sure whom to add. Will remove your name. @kevinlee12 Are you ok with this?

Reply:
that is fine.